crates/rethnet_rpc_server/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rethnet_rpc_server"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.1.0-dev"
```

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/3998/commits/3221c121a58a8833cf2c5540cc0e0deb0b551244